### PR TITLE
fix(tier4_autoware_utils): fix covariance enum name

### PR DESCRIPTION
**Pull request by scepter914**
## Description

This PR change from pose_covariance_index::POSE_COV_IDX to xyzrpy_covariance_index::XYZRPY_COV_IDX.
This change of util library may make easy to read.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp [hunk 1]
```diff
@@ -162,7 +162,7 @@ TrackedObjects RadarTracksMsgsConverterNode::convertRadarTrackToTrackedObjects()
   TrackedObjects tracked_objects;
   tracked_objects.header = radar_data_->header;
   tracked_objects.header.frame_id = node_param_.new_frame_id;
-  using P_IDX = tier4_autoware_utils::pose_covariance_index::POSE_COV_IDX;
+  using C_IDX = tier4_autoware_utils::xyzrpy_covariance_index::XYZRPY_COV_IDX;
   using R_IDX = tier4_autoware_utils::xyz_upper_covariance_index::XYZ_UPPER_COV_IDX;
```
Comment (kenji-miyake) [inline perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp:166]: What are `C_` and `R_`?

Comment (scepter914) [reply perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp:166]: I mean Covariance index and Radar covariance index, but I think it is not suitable abbreviation.
Do you have any suggestion?

Comment (kenji-miyake) [reply perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp:166]: I'd use `POSE_IDX`, `RADER_IDX` or `POSE_COV_IDX`, `RADER_COV_IDX`.
And rename variables:
- covariance -> pose_cov
- radar_track.position_covariance -> radar_cov

**Review by kenji-miyake — APPROVED** [review]
Looks clean! Thanks.

**Pull request merged**